.github/scripts/transform_dbt_test_results.py

```diff
@@ -108,8 +105,9 @@
 CLASS_FIELD = "class"
 WHO_FIELD = "who"
 WEN_FIELD = "wen"
-# Overrides for default display names for dbt tests
-CUSTOM_TEST_NAMES = {
+# Mapping that defines category names that should be reported for tests
+# based on their generics
+TEST_CATEGORIES = {
```

Comment:
This variable previously had a confusing name, and I think `TEST_CATEGORIES` is clearer.